<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExMulticast</class>
 <widget class="QMainWindow" name="ExMulticast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExMulticast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>组播端口:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxBind">
          <property name="maximum">
           <number>65000</number>
          </property>
          <property name="value">
           <number>8888</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>组播地址：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxIp">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>239.255.43.21</string>
          </property>
          <item>
           <property name="text">
            <string>239.255.43.21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>239.255.100.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>239.255.255.100</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>组播发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string>239.0.0.0～239.255.255.255为本地管理组播地址范围</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="actStop"/>
   <addaction name="separator"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image4.png</normaloff>:/images/Image4.png</iconset>
   </property>
   <property name="text">
    <string>加入</string>
   </property>
   <property name="toolTip">
    <string>加入组播</string>
   </property>
  </action>
  <action name="actStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image5.png</normaloff>:/images/Image5.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出组播</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image1.png</normaloff>:/images/Image1.png</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空文本内容</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Image2.jpg</normaloff>:/images/Image2.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
